espnet2/bin/enh_scoring.py

```diff
@@ -223,17 +224,35 @@ def scoring(
                 if pesq:
                     if sample_rate == 8000:
                         mode = "nb"
+                        ref_ = ref[i]
+                        inf_ = inf[int(perm[i])]
                     elif sample_rate == 16000:
                         mode = "wb"
+                        ref_ = ref[i]
+                        inf_ = inf[int(perm[i])]
+                    elif sample_rate > 16000:
+                        mode = "wb"
```

Comment:
We'd better add some log or warning here.

Reply:
OK!